### ISSUE TITLE
Missing apple touch icon

### ISSUE THREAD

**Issue by prajwalkulkarni** [codeisscience/journal-policy-tracker-frontend#153]
Thank you for taking the time to file a Bug report.

**Describe the bug**
The `index.html` specifies a link tag to a logo that'll be shown on the home screen in iOS devices when a user adds this website to their home screen. However, this logo is missing. This would force the system to capture the home screen and show it is a logo instead. 

**To Reproduce**

1. Fire up the dev server
2. open the website
3. Inspect and head to the console tab
4. See the error

**Expected behavior**
A logo should be added with a resolution of 144*144px or more, or logos of multiple dimensions can be added if targeting multiple iOS devices.

**Error**
>Error while trying to use the following icon from the Manifest: http://localhost:3000/logo192.png (Download error or resource isn't a valid image)

**Desktop (please complete the following information):**

- OS: Windows 11 (Error seen on console, but the actual error is reflected on iOS devices)
- Browser: Error seen on chrome's console, reflected in Safari/iOS home screen
- Version: 100.0

**Smartphone (please complete the following information):**

- Device: iPhone
- OS: iOS 15.3
- Browser: Safari

**Comment by prajwalkulkarni** [comment]
@padmajabhol Kindly let me know if I can work on fixing this.

**Comment by padmajabhol** [comment]
I might need @ikayz approval on this since I'm not very sure

**Comment by ikayz** [comment]
@prajwalkulkarni go ahead with this issue

**Comment by prajwalkulkarni** [comment]
@ikayz @padmajabhol @yochannah , I've linked a PR that adds the apple-touch-icon, I've also removed extraneous non-code content from `Contact.js` that came from PR#125, apart from that there was also a linting error about the component being an arrow function, I've addressed and fixed that as well(as the latter two changes did not involve many changes, I've added them to this commit). Kindly review and let me know if there are any other changes to be made. Thanks!
